app/i18n/pt.json

```diff
@@ -339,6 +348,15 @@
         "delete_api_key": "Apagar chave API",
         "csv_export": "Exportação CSV"
       }
+    },
+    "specialVoting": {
+      "title": "",
+      "ended": "",
+      "link": "",
+      "collectively": "",
+      "proportionally": "",
+      "explanation": "",
+      "description": ""
```

Comment:
Empty values for pt?

Reply:
uff… I need to make that pre-comit script
But in any case, pt is not an officially supported language.

Reply:
Please add the PT translations before merging.